src/templates/teampage.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { GatsbyImage } from 'gatsby-plugin-image';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import MemberDescription from '../components/MemberDescription';
+import { memberData } from '../data/mockMemberData';
+
+const TeamPage = ({ pageContext: { teamPageData } }) => (
+    //TODO[CHRIS FROM YSSA]: ^^ about page data will need to be changed to utilize member data
+    <>
+        <Header />
```

Comment:
Had to import Header and Footer individually versus using the Layout Component to fit the look of the mocks

Reply:
ah okay, we should probably get the designs for each page layout to follow some sort of format so we don't have to import them separately and have the padding not be unified between pages. We can def leave it for now and investigate later.